mysql-test/columnstore/basic/r/mcs84_rollup.result

```diff
@@ -16,9 +16,35 @@ year	SUM(sales)
 SELECT year, SUM(sales) FROM booksales GROUP BY year WITH ROLLUP ORDER BY year;
 ERROR HY000: Incorrect usage of CUBE/ROLLUP and ORDER BY
 SELECT year, SUM(sales) FROM booksales GROUP BY year WITH ROLLUP;
```

Comment:
I suggest to extend the test case with at least wide decimal and dictionary-based string types.